spec/javascript-spec.coffee

```diff
@@ -757,6 +788,31 @@ describe "JavaScript grammar", ->
       expect(tokens[10]).toEqual value: '}', scopes: ['source.js', 'meta.import.js', 'punctuation.definition.modules.end.js']
       expect(tokens[12]).toEqual value: 'from', scopes: ['source.js', 'meta.import.js', 'keyword.control.js']
 
+      lines = grammar.tokenizeLines """
+        import
+        \\t
```

Comment:
I clearly can't edit a language grammar while compiling a V8 build, it seems. )))))))))))

Pushed an amended commit.